src/common/xo/index.js

```diff
@@ -634,6 +634,60 @@ export const editNetwork = ({ id }, props) => (
   call('network.set', { ...props, id })
 )
 
+import CreateNetworkModalBody from './create-network-modal'
+export const createNetwork = container => (
+  confirm({
+    icon: 'network',
+    title: _('newNetworkCreate'),
+    body: <CreateNetworkModalBody container={container} />
+  }).then(
+    params => xo.call('network.create', params),
+    noop
+  )
+)
+
+export const deleteNetwork = ({ network }) => (
```

Comment:
The param should be either a network object or a network id, use `resolveId()` for that.